.gitignore

```diff
@@ -72,5 +72,6 @@ coverage.xml
 # Front-End #
 #############
 node_modules/
+lib/
```

Comment:
Which do you think is preferable? 

I've been following [this](http://www.polyglotweekly.com/2015/11/29/writing-your-first-es2015-javascript-module.html) guide btw.